stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1144,8 +1144,8 @@
 (define-public (delegate-stack-extend
                     (stacker principal)
                     (pox-addr { version: (buff 1), hashbytes: (buff 32) })
-                    (signer-key (buff 33))
-                    (extend-count uint))
+                    (extend-count uint)
```

Comment:
This change should be a different PR related to this issue: https://github.com/stacks-network/stacks-core/issues/4230

Reply:
I already address it here & wrote tests for it accordingly. I'd rather update the description to address #4230 as well then revert the changes across all files. 

That work?

Reply:
The issue does not contain a preferred solution, still unclear what/whether should be implemented.

Reply:
Addressed & reverted change.